syntax = "proto3";

package pb;

message Ledger
{
    double amount=1;
    bool successful=2;
    string transaction_dt=3;
}

message Account
{
    string full_name=1;
    string social_security=2;
    string date_created_dt=3;
    double balance=4;
    Ledger ledger=5;

}

message Accounts
{
    map<uint64, Account> accounts = 1;
}
